Refreshed content - removed Alexander, updated nominations (#211)
w3f/polkadot-wiki: docs/maintain-guides-how-to-nominate-kusama.md

Pick "Account Actions", then click the blue "New Stake" button.

You will see a modal window that looks like the below:

![Bonding](/img/NPoS/nominate2.png)

Select a "value bonded" that is **less** than the total amount of KSM you have, so you have some left over to pay transaction fees. Be mindful of the reaping threshold - the amount that must remain in an account lest it be burned. That amount is 0.01 in Kusama, so it's recommended to keep at least 0.1 KSM in your account to be on the safe side.

Choose whatever payment destination sounds good to you. If you're unsure choose "Stash account (increase amount at stake)".

### Step 2: Nominate a validator

> You are now bonded. Being bonded means your tokens are unlocked for slashing if the validators you nominate misbehave. Any bonded funds can now be distributed to up to 16 validators. Be careful about the validators you choose since you will be slashed if your validator commits an offence.

Reviewer: ```suggestion
You are now bonded. Being bonded means your tokens are locked and  could be slashed if the validators you nominate misbehave. All bonded funds can now be distributed to up to 16 validators. Be careful about the validators you choose since you will be slashed if your validator commits an offence.
```

Reviewer: Can we add a link to the slashing page here for those who want to learn more about the offences?

Author: ```suggestion
You are now bonded. Being bonded means your tokens are locked and could be [slashed](learn-staking#slashing) if the validators you nominate misbehave. Any bonded funds can now be distributed to up to 16 validators. Be careful about the validators you choose since you will be slashed if your validator commits an offence.
```
